stablehlo/dialect/VhloBase.td

```diff
@@ -34,6 +34,49 @@ def VersionedInterface : OpInterface<"VersionedInterface"> {
   ];
 }
 
+def VHLO_VersionedAttrInterface : AttrInterface<"VersionedAttrInterface"> {
+  let cppNamespace = "::mlir::vhlo";
+  let methods = [
+    InterfaceMethod<
+      "Returns the minimum version an attribute is supported in.",
+      "mlir::vhlo::Version", "getMinVersion">,
+    InterfaceMethod<
+      "Returns the maximum verison an attribute is supported in.",
```

Comment:
Typo: "version". (Here, below and above).

Reply:
Fixed. Was also the same typo in `STABLEHLO_CUSTOM_CALL_API_VERISON_UNSPECIFIED`, fixed that too.